### DISCUSSION TITLE
Remove/change the Debug logging

### DISCUSSION THREAD

**Discussion by wiverson** [supabase/gotrue-csharp#54] [Ideas]
Right now there are eight places in the client code that write to Debug. I think these should be removed and replaced with slightly different options:

- ExceptionHandler.cs - remove
- Client.cs - replace with an (optional) debug notification callback

The test suite could add the debug callback notification to mimic existing behavior.

LMK if ok to send a PR.

**Comment by acupofjose** [comment]
Sure get in there!

**Comment by wiverson** [comment]
Resolved over recent PRs including.

Woot!
